project/core/settings.py

```diff
@@ -156,7 +156,7 @@
     )
 
     DEFAULT_AUTHENTICATION_CLASSES = (
-        "api.authentication.CsrfExemptSessionAuthentication",
+        "accounts.authentication.CsrfExemptSessionAuthentication",
```

Comment:
Add a note to deprecate this authentication method and follow DRF CSRF guidelines.
```suggestion
        # TODO: deprecate this authentication method and follow DRF CSRF guidelines
        # https://www.django-rest-framework.org/topics/ajax-csrf-cors/
        "accounts.authentication.CsrfExemptSessionAuthentication",
```